lib/IOExt/IOExt.cpp

```diff
@@ -25,22 +19,10 @@ void IOExt::init() {
   // CRITICAL SECTION I2C: start
   xSemaphoreTake(i2cBus.mutex, portMAX_DELAY);
 
-  // setup pins
-  IOExt2.pinMode(P0, INPUT); // left indicator
-  IOExt2.pinMode(P1, INPUT); // right indicator
-  IOExt2.pinMode(P2, INPUT); // set tempomat/poweromat
-  IOExt2.pinMode(P3, INPUT); // set tempo or power mode
-  IOExt2.pinMode(P4, INPUT); // horn
-  IOExt2.pinMode(P5, INPUT); // position lights
-  IOExt2.pinMode(P6, INPUT); // driving lights
-  IOExt2.pinMode(P7, INPUT); // next screen
-
-  // start
-  if (IOExt2.begin()) {
-    printf("[v] IOExt2 inited.\n");
-  } else {
-    printf("[x] IOExt2 init failed.\n");
-    // TODO: action for init error?
+  for(int i = 0; i < PCF8574_NUM_DEVICES; i++){
+    if(IOExt[i].begin()){
+      printf("Initialization of IOExt0%u failed.\n", i);
+    }
```

Comment:
This format is for the boot screen importend to see what works and what fails:
  printf("[v] IOExt2 inited.\n");
  } else {
    printf("[x] IOExtXXX init failed.\n");
    // TODO: action for init error?

Reply:
Good point, has been fixed.